### PR TITLE
Show preview in prepare condition

### LINKED ISSUE #350: [FE] Display Condition ID once required fields are filled
When creating a condition, we can know what the condition ID is as soon as the question ID, outcomes, and reporting address have been filled.

We should:
1. display the resulting ID
2. Disable the prepare button if the condition already exists
3. Provide a link to the condition if it already exists

### PR DESCRIPTION
**Pull request by hpmaxi**
Closes #350 
Captura de pantalla de 2020-10-21 18-17-00
Captura de pantalla de 2020-10-21 18-18-58
Captura de pantalla de 2020-10-21 18-19-05

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr498--conditionaltokensexplorer.review.gnosisdev.com

**Comment by elena-zh** [comment]
Wow! Great!

File: src/pages/PrepareCondition/index.tsx [hunk 1]
```diff
@@ -474,8 +491,11 @@ export const PrepareCondition = () => {
           )}
           {isConditionAlreadyExist && (
             <StatusInfoInline status={StatusInfoType.warning}>
-              Condition already exist. Please use another question ID or change the number of
-              outcomes.
+              Condition{' '}
+              <Link href={`#/conditions/${conditionIdPreview}`} target="_blank">
+                {conditionIdPreview}
```
Comment (mariano-aguero) [inline src/pages/PrepareCondition/index.tsx:496]: Maybe we can use the shorter version for a hash with dots ? Looks really long and sometimes breaks the `word`
Screenshot_20201022_091732

Comment (gabitoesmiapodo) [reply src/pages/PrepareCondition/index.tsx:496]: Agreed.

**Review by mariano-aguero — APPROVED** [review]
I left a comment, the rest looks good to me . Maybe the `Success` title can be removed (with a conditional, so the rest of the application is not affected) , the hash in the success box is really long too but, what we can do about it ? @gabitoesmiapodo thoughts ?
I approve it, so we can move faster, let's wait for Gabriel review

**Review by gabitoesmiapodo — CHANGES_REQUESTED** [review]
Looks great.

A few issues:

- Both info blocks should be at the bottom (after the last input field). We expect the users to fill the inputs from top to bottom, so that's where they will be when they finish filling everything (as a side effect, you could use the component for warning / success only once).
- You could use the "cut in the middle" version of the condition's id (not a strong opinion here), and add a copy button (there's a component to do that IIRC, it comes handy wether we cut the id or not). 
- We should show a spinner while fetching / calculating the preview.

**For the info blocks:**

Warning text:

- Title: Duplicated Condition
- Text: Condition `0xe3ffbe...3a5aaa` (*) already exist(**). Please use another question Id or a different number of outcomes.

(*) add link to condition (same window and `target="_blank"` and copy button, I think there's a component that does that.
(**) typo, it should be "exists".

Success text:

- Title: Condition Id Preview
- Text: The condition's ID + copy button.

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr498--conditionaltokensexplorer.review.gnosisdev.com

**Review by gabitoesmiapodo — APPROVED** [review]

**Comment by elena-zh** [comment]
Verified. Looks and works fine. 
Issue with the duplicated condition link is found: #522

**Pull request merged**